randovania/generator/generator.py

```diff
@@ -333,3 +345,32 @@ def actual_status_update(msg: str) -> None:
             )
 
     return result
+
+
+def force_logical_placement(
+    pickups: list[PickupEntry], game: GameDescription, lpp_config: LogicalPickupPlacementConfiguration
+) -> Requirement:
+    """
+    Creates a Requirement with the game's victory condition and the selected pickup set.
```

Comment:
not quite a fan of the "and the selected pickup set".
Maybe more along the lines of "Returns a game's victory condition as a requirement adjusted for a given LogicalPickupPlacementConfiguration"?